.travis.yml

```diff
@@ -2,9 +2,12 @@ language: ruby
 cache: bundler
 rvm:
   - 1.9.3
```

Comment:
It doesn't hurt to leave it in _for now_, as EOL is Feb 27. We can remove it after that.